test_project/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,18 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Django command to pause execution until database is available"""
+    def handle(self, *args, **options):
+        self.stdout.write('Waiting for database...')
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections['default']
+            except OperationalError:
+                self.stdout.write('Database unavailable, waiting 1 second...')
+                time.sleep(1)
+        self.stdout.write(self.style.SUCCESS('Database available!'))
```

Comment:
Nice, but this should have a time limit, otherwise it would be infinite.

Reply:
I removed the wait_for_db command because I was trying to add the timeout but TimeoutError isn't available in Python 2.7. I added a script file and now I wait for the database using wait-for-it.sh, I downloaded it from https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh, I don't know if I should add a link to that repository.

Reply:
Well, you could have break the while after a number of cycles, but the wait-for-it script is ok, we already have used it before.